src/features/memoroom/MemoRoom.js

```diff
@@ -78,6 +104,27 @@ function MemoRoom() {
     memoTagInfo[memoId] = memoInfo.tags.join(",");
   });
 
+  function handleShareButtonClick() {
+    setIsShareModalOpen(!isShareModalOpen);
+  }
+
+  function handleInviteMailSubmit(event) {
+    event.preventDefault();
+
+    const { email } = event.target;
+    const participant = Object.entries(participants).find(([id, data]) => {
+      return email.value === data.email;
+    });
+
+    if (!participant) {
+      dispatch(postSendMailRequest({ userId, memoroomId, email: email.value }));
+    }
+
+    if (participant) {
+      setErrorMessage("❗️ Already participated member");
+    }
```

Comment:
```suggestion
    if (!participant) {
      dispatch(postSendMailRequest({ userId, memoroomId, email: email.value }));
      return;
    }
    
    setErrorMessage("❗️ Already participated member");
```
early return으로 수정해도 좋을 것 같습니다.

Reply:
넵!